lxd/container_lxc.go

```diff
@@ -1921,6 +1921,16 @@ func (c *containerLXC) initLXCIPVLAN(cc *lxc.Container, networkKeyPrefix string,
 	}
 
 	if m["ipv4.address"] != "" {
+		//Check necessary sysctls are configured for use with l2proxy parent in IPVLAN l3s mode.
+		ipv4FwdPath := fmt.Sprintf("ipv4/conf/%s/forwarding", m["parent"])
+		sysctlVal, err := networkSysctlGet(ipv4FwdPath)
+		if err != nil {
+			return fmt.Errorf("Error reading net sysctl %s: %s", ipv4FwdPath, err)
```

Comment:
For those cases, we prefer the use of errors.Wrap these days (trying to use it for new ones and slowly converting some of the old ones as we go)

Reply:
Main benefit of that over fmt.Errorf is that you can get to the underlying error struct

Reply:
@stgraber cool will update, I read Dave Cheney's post about this in 2016 (https://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully). I hope this makes it into the std lib one day.